content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -136,6 +136,35 @@ The following table shows the required settings for many common Kubernetes envir
 | Red Hat OpenShift 3.10+ | _(none)_ | _(none)_ |
 | Red Hat OpenShift 4.2+ | `--set components.cni.namespace=kube-system --set values.cni.cniBinDir=/var/lib/cni/bin --set values.cni.cniConfDir=/etc/cni/multus/net.d --set values.cni.chained=false --set values.cni.cniConfFileName="istio-cni.conf" --set values.sidecarInjectorWebhook.injectedAnnotations."k8s\.v1\.cni\.cncf\.io/networks"=istio-cni` | _(none)_ |
 
+#### Instructions for Istio 1.4.x and OpenShift
+
+Due to a [limitation](https://github.com/istio/istio/issues/19196) in `istioctl` 1.4.x, you will need to create a file and
+pass that file to `istioctl`. The documentation for Istio 1.4.x is out of date for OpenShift installation.
+
+{{< text yaml >}}
+cat <<'EOF' > cni-annotations.yaml
```

Comment:
Did you actually try these instructions? I get this error: `could not unmarshal the overlay file: unknown field "components" in v1alpha2.IstioControlPlaneSpec`

This file works with `istioctl manifest`:
```yaml
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  cni:
    enabled: true
    components:
      cni:
        namespace: kube-system
  values:
    cni:
      chained: false
      cniBinDir: /var/lib/cni/bin
      cniConfDir: /etc/cni/multus/net.d
      cniConfFileName: istio-cni.conf
    sidecarInjectorWebhook:
      injectedAnnotations:
        "k8s.v1.cni.cncf.io/networks": istio-cni
```

Reply:
My mistake. Turns out I had istioctl 1.5 in my path. Updated. I'll verify again before it goes live.